server/internal/infrastructure/fs/file.go

```diff
@@ -124,6 +125,14 @@ func (f *fileRepo) GetURL(a *asset.Asset) string {
 	return f.urlBase.JoinPath(assetDir, fileUUID[:2], fileUUID[2:], url.PathEscape(a.FileName())).String()
 }
 
+func (f *fileRepo) IssueUploadAssetLink(ctx context.Context, filename string, expiresAt time.Time) (string, string, error) {
+	return "", "", gateway.ErrUnsupportedOperation
```

Comment:
Considering that it is OSS and will run in a variety of environments, it would be great if the upload URL could be used for file systems as well. However, this could be done in a separate PR or implemented by someone else.